department-of-veterans-affairs/abd-vro
# Add cancellation as claim note through VRO's svc-bgs-api

## What was the problem?

EP Merge needs to add a claim note to the EP400 upon canceling the claim. 

Associated tickets or Slack threads:
- #2161 
- Stacked on PR:
   - #2303 


## How does this fix it?[^1]

Adds step in state machine to add a claim note with the cancellation reason to the EP400 claim. 

## How to test this PR
- pull code
- from /domain-ee/ee-ep-merge-app run pytest
- from /domain-ee/ee-ep-merge-app run pytest ./integration
- run this [CI](https://github.com/department-of-veterans-affairs/abd-vro/actions/workflows/ee-ep-merge-end-to-end.yml) action to verify successful integration test


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
